packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -438,9 +453,22 @@ class SliverGeometry extends Diagnosticable {
   /// A sliver that occupies no space at all.
   static const SliverGeometry zero = const SliverGeometry();
 
-  /// The (estimated) total scroll extent that this sliver has content for. In
-  /// other words, the scroll offset of the end of the last bit of content of
-  /// this sliver.
+  /// The (estimated) total scrollable extent that this sliver has content for.
+  ///
+  /// It's the amount of scrolling the user needs to do to get from the
+  /// beginning of this sliver to the end of this sliver past a point in a
+  /// viewport.
```

Comment:
the grammar of that sentence is weird. maybe by "past" you mean "at"? though even then it's a bit weird.

Reply:
Removed the last bit. It was superfluous